cortx-s3samplecode/node.js/cortx-s3samplecode-node.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Seagate/cortx/pullRequest?prid=8060152)
```suggestion
"use strict";
```